CharlieBackend.Business/Services/Interfaces/IEventsService.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using CharlieBackend.Core.DTO.Schedule;
+using CharlieBackend.Core.Models.ResultModel;
+
+namespace CharlieBackend.Business.Services.Interfaces
+{
+    public interface IEventsService
+    {
+        public Task<Result<ScheduledEventDTO>> UpdateScheduledEventByID(long scheduledEvendID, UpdateScheduledEventDto scheduleModel);
```

Comment:
Rename UpdateScheduledEventByID to UpdateAsync, DeleteConcreteScheduleByIdAsync to DeleteAsycn, GetConcreteScheduleByIdAsync to GetAsycn

Reply:
Change signature of returning result type from Task<Result<ScheduledEventDTO>> to Task<ScheduledEventDTO>

Reply:
Renamed. 
Signature has been changed for the methods get and update, but for deleting I should use:
Result<bool>.GetSuccess(true);
So I leave this part of the code (public Task<Result<bool>> DeleteAsync(long id);).